experiments/neural_mcts/run_like.py

```diff
@@ -0,0 +1,134 @@
+import ast
+import pathlib
+import time
+import subprocess
+import sys
+
+import pypolygames
+import tube
+import polygames
+import mcts
+import pypolygames.utils as utils
+
+"""
+This file is used to run a model with exactly the same hyperparameters as a given model.
+"""
+
+
+# Check to make sure we have exactly two arguments
+if len(sys.argv) < 2:
```

Comment:
I recommend using `argparse`. It's very nice to use with Copilot (because it does most of the boilerplate for you). Nevertheless this is fine.

Reply:
Suggested rectification: just thumbs up and move on